includes/adapter/OrderLineFromWooCommerce.php

```diff
@@ -29,8 +29,9 @@ public function __construct(WC_Order_Item $wcOrderItem)
     protected function prepareItemData(WC_Order_Item $wcOrderItem): array
     {
         $wcItemData = $wcOrderItem->get_data();
+        $wcProduct  = $wcOrderItem->get_product();
```

Comment:
Dit is zo'n plek waar product null kan zijn wanneer het deleted is, graag rekening mee houden.

Reply:
Na het kijken naar mijn eigen pr, zou ik denken dit mogelijk anders op te lossen, zodat er altijd een product doorgegeven wordt en een prijs, omdat dat wel essentieel is in dit geval, door zelf de orderline prijs (en evt. vat) terug te rekenen op basis van quantity, zodat de api het weer kan vermenigvuldigen. Dus ik zou doen:
`$price = ($wcItemData[' subtotal'] * 100 / $wcOrderItem->get_quantity());`
Ofzoiets maar dan mooi. En net zoiets voor de vat. Dan ben je voor deze essentiele regel niet afhankelijk van wel of niet bestaan van het product.